src/mpid/ch3/channels/nemesis/netmod/tcp/tcp_getip.c

```diff
@@ -75,9 +75,12 @@ int MPIDI_GetIPInterface( MPIDI_CH3I_nem_tcp_ifaddr_t *ifaddr, int *found )
 
     MPIR_FUNC_VERBOSE_ENTER(MPID_STATE_MPIDI_GETIPINTERFACE);
     *found = 0;
-
+#if defined(AF_INET6)
+	fd = socket(AF_INET6, SOCK_DGRAM, 0);
```

Comment:
Looks like hard tab is used here. MPICH standard is to use 4 space characters for a tab.

Reply:
Hi, Hajimefu, 
Already checked in commit replacing all hard tabs with 4  space characters.